rewards/classes/RewardsManager.py

```diff
@@ -25,12 +25,13 @@
 
 
 class RewardsManager:
-    def __init__(self, chain: str, cycle: int, start: int, end: int):
+    def __init__(self, chain: str, cycle: int, start: int, end: int, boosts):
```

Comment:
type hint on boosts

Reply:
I think we should add custom typing for the boosts/tree data in a separate PR and leave this for now

Reply:
ok good idea i'll make a ticket

Reply:
https://github.com/Badger-Finance/badger-rewards/issues/139